redwood/src/lib.rs

```diff
@@ -94,6 +98,19 @@ pub fn generate_source_key_pair(
     Ok((public_key, secret_key, format!("{}", cert.fingerprint())))
 }
 
+#[pyfunction]
+pub fn is_valid_public_key(input: &str) -> Result<String> {
+    let policy = &StandardPolicy::new();
```

Comment:
(Very minor point but) if the code becomes more complex or we ever stop using StandardPolicy, we may want to create one policy instance and pass it to `encrypt` and `is_valid_pubkey` to ensure there's no developer error and we're always testing and encrypting against the same policy.

Reply:
That's a good point, and since you mentioned it, I looked, `StandardPolicy::new()` is const, so we can in fact already do that! Commit incoming...